build.fsx

```diff
@@ -93,7 +93,36 @@ Target "RunTests" (fun _ ->
                 { p with 
                     WorkingDir = (Directory.GetParent project).FullName
                     TimeOut = TimeSpan.FromMinutes 30. })
-                (sprintf "xunit -parallel none -teamcity -xml %s_xunit.xml" (outputTests @@ fileNameWithoutExt project)) 
+                (sprintf "xunit -parallel none -teamcity -f net452 -c Release -xml %s_xunit.xml" (outputTests @@ fileNameWithoutExt project)) 
+
+    CreateDir outputTests
+    projects |> Seq.iter (runSingleProject)
+)
+
+Target "RunTestsNetCore" (fun _ ->
+    let projects =
+        match isWindows with
+        // Windows
+        | true -> !! "./**/core/**/*.Tests.csproj"
+                  ++ "./**/contrib/**/*.Tests.csproj"
+                  -- "./**/Akka.Remote.TestKit.Tests.csproj"
+                  -- "./**/Akka.MultiNodeTestRunner.Shared.Tests.csproj"
+                  -- "./**/serializers/**/*Wire*.csproj"
+        // Linux/Mono
+        | _ -> !! "./**/core/**/*.Tests.csproj"
+                  ++ "./**/contrib/**/*.Tests.csproj"
+                  -- "./**/serializers/**/*Wire*.csproj"
+                  -- "./**/Akka.Remote.TestKit.Tests.csproj"
+                  -- "./**/Akka.MultiNodeTestRunner.Shared.Tests.csproj"
+                  -- "./**/Akka.API.Tests.csproj"
```

Comment:
Can you explain?  We have been running .NET Core on both platforms, just 1 Platform + 2 Frameworks.  My goal is to run 1 Platform + 1 Framework for our intended supported Frameworks.

So instead of
```
Windows
  .NET Core
  .NET 4.5
Linux/Mono
  .NET 4.5
  .NET Core
```
We'll be doing
```
Windows
  .NET Core
Windows
  .NET 4.5
Linux/Mono
  .NET 4.5
Linux
  .NET Core
```